### PR TITLE
Terrain macro material component

### PR DESCRIPTION
**Pull request by sphrose**
Signed-off-by: sphrose <82213493+sphrose@users.noreply.github.com>

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/user-guide/components/reference/_index.md [hunk 1]
```diff
@@ -209,8 +209,9 @@ The components below are grouped by type as they appear in the O3DE Editor.
 
 | Component | Description | 
 | - | - |
-| [Terrain Layer Spawner](/docs/user-guide/components/reference/terrain/layer_spawner/) | Spawns a terrain region contained within configurable bounds, and allows prioritization of overlapping terrain layers. |
-| [Terrain Height Gradient List](/docs/user-guide/components/reference/terrain/terrain_height_gradient_list.md) | Provides terrain height data from a list of gradients. |
+| [Terrain Layer Spawner](/docs/user-guide/components/reference/terrain/layer_spawner) | Spawns a terrain region contained within configurable bounds, and allows prioritization of overlapping terrain layers. |
+| [Terrain Height Gradient List](/docs/user-guide/components/reference/terrain/terrain_height_gradient_list) | Provides terrain height data from a list of gradients. |
+| [Terrain Macro Material](/docs/user-guide/components/reference/terrain/terrain-macro-material.md) | Provides a macro level method of defining the appearance of a region of terrain. |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/_index.md:214]:
```suggestion
| [Terrain Macro Material](/docs/user-guide/components/reference/terrain/terrain-macro-material) | Provides a macro level method of defining the appearance of a region of terrain. |
```

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+linktitle: Terrain Macro Material
+description: 'Open 3D Engine (O3DE) Terrain Macro Material component reference.'
+weight: 100
+---
+
+The **Terrain Terrain Macro Material** component provides a method of defining a macro-level appearance of a region of terrain.
+
+## Usage ##
+
+You assign a color and a normal texture by dragging texture assets to the **Color Texture** or **Normal Texture** fields, or by clicking {{< icon "file-folder.svg" >}}.
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:12]:
```suggestion
You assign a color and a normal texture by dragging texture assets to the **Color Texture** or **Normal Texture** fields, or by clicking {{< icon "file-folder.svg" >}}. Once the textures are assigned, you can adjust the appearance by using the **Normal Flip X** and **Normal Flip Y** toggles, and adjust the strength of the normals by using the **Normal Factor** slider. You can configure the dimensions of the region by adjusting the [Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape).
```
Linebreak fix.

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+Once the textures are assigned, you can adjust the appearance by using the **Normal Flip X** and **Normal Flip Y** toggles, and adjust the strength of the normals by using the **Normal Factor** slider.
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:13]:
```suggestion
```

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+weight: 100
+---
+
+The **Terrain Terrain Macro Material** component provides a method of defining a macro-level appearance of a region of terrain.
+
+## Usage ##
+
+You assign a color and a normal texture by dragging texture assets to the **Color Texture** or **Normal Texture** fields, or by clicking {{< icon "file-folder.svg" >}}.
+Once the textures are assigned, you can adjust the appearance by using the **Normal Flip X** and **Normal Flip Y** toggles, and adjust the strength of the normals by using the **Normal Factor** slider. 
+You can configure the dimensions of the region by adjusting the [Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape)
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:14]:
```suggestion
```

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+[Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape)
+
+## Properties ##
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Color Texture** | The image that will be rendered on the terrain. | Texture | None |
+| **Normal Texture** | The texture that will be used as the normal map. | Texture | None |
+| **Normal Flip X** | Set to true to flip the normals about X. | bool | false |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:30]:
```suggestion
| **Normal Flip X** | Set to true to flip the normals about X. | Boolean | `False` |
```
Suggestion to match formatting of other component pages.

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+| **Normal Flip Y** | Set to true to flip the normals about Y. | bool | false |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:31]:
```suggestion
| **Normal Flip Y** | Set to true to flip the normals about Y. | Boolean | `False` |
```

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+| **Normal Factor** | Adjusts the strength of the normal values. | float | 1.0 |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:32]:
```suggestion
| **Normal Factor** | Adjusts the strength of the normal values. | Float | `1.0` |
```

File: content/docs/user-guide/gems/reference/environment/terrain/_index.md [hunk 8]
```diff
@@ -30,4 +30,5 @@ To enable the Terrain Gem, do the following:
 | Component | Description | 
 | - | - |
 | [Terrain Layer Spawner](/docs/user-guide/components/reference/terrain/layer_spawner.md) | Spawns a terrain region contained within configurable bounds, and allows prioritization of overlapping terrain layers. |
-| [Terrain Height Gradient List](/docs/user-guide/components/reference/terrain/terrain_height_gradient_list.md) | Provides terrain height data from a list of gradients. |
\ No newline at end of file
+| [Terrain Height Gradient List](/docs/user-guide/components/reference/terrain/terrain_height_gradient_list.md) | Provides terrain height data from a list of gradients. |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/gems/reference/environment/terrain/_index.md:33]:
```suggestion
| [Terrain Height Gradient List](/docs/user-guide/components/reference/terrain/terrain_height_gradient_list) | Provides terrain height data from a list of gradients. |
```

File: content/docs/user-guide/gems/reference/environment/terrain/_index.md [hunk 9]
```diff
@@ -30,4 +30,5 @@ To enable the Terrain Gem, do the following:
...
+| [Terrain Macro Material](/docs/user-guide/components/reference/terrain/terrain-macro-material.md) | Provides a macro level method of defining the appearance of a region of terrain. |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/gems/reference/environment/terrain/_index.md:34]:
```suggestion
| [Terrain Macro Material](/docs/user-guide/components/reference/terrain/terrain-macro-material) | Provides a macro level method of defining the appearance of a region of terrain. |
```

**Review by FiniteStateGit — CHANGES_REQUESTED** [review]

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+
+
+## MacroMaterialData
+
+This structure is used when sending out information about the macro material settings
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:37]:
```suggestion
This structure is used when sending out information about the macro material settings.
```

**Review by FiniteStateGit — CHANGES_REQUESTED** [review]

File: content/docs/user-guide/components/reference/_index.md (hunk 1 above)
Comment (sphrose) [reply content/docs/user-guide/components/reference/_index.md:214]: Fixed

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md (hunk 2 above)
Comment (sphrose) [reply content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:12]: Done

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md (hunk 5 above)
Comment (sphrose) [reply content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:30]: Done

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md (hunk 6 above)
Comment (sphrose) [reply content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:31]: Done

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md (hunk 7 above)
Comment (sphrose) [reply content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:32]: Done

File: content/docs/user-guide/gems/reference/environment/terrain/_index.md (hunk 8 above)
Comment (sphrose) [reply content/docs/user-guide/gems/reference/environment/terrain/_index.md:33]: Done

Comment (sphrose) [reply content/docs/user-guide/gems/reference/environment/terrain/_index.md:33]: Done

File: content/docs/user-guide/gems/reference/environment/terrain/_index.md (hunk 9 above)
Comment (sphrose) [reply content/docs/user-guide/gems/reference/environment/terrain/_index.md:34]: Done

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md (hunk 10 above)
Comment (sphrose) [reply content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:37]: Done

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+[Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape)
+
+## Properties ##
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Color Texture** | The image that will be rendered on the terrain. | Texture | None |
+| **Normal Texture** | The texture that will be used as the normal map. | Texture | None |
+| **Normal Flip X** | Set to true to flip the normals about X. | Boolean | false |
+| **Normal Flip Y** | Set to true to flip the normals about Y. | Boolean | false |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:29]:
```suggestion
| **Normal Flip Y** | Set to true to flip the normals about Y. | Boolean | `False` |
```

**Review by FiniteStateGit — CHANGES_REQUESTED** [review]
Minor edits.

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+linktitle: Terrain Macro Material
+description: 'Open 3D Engine (O3DE) Terrain Macro Material component reference.'
+weight: 100
+---
+
+The **Terrain Terrain Macro Material** component provides a method of defining a macro-level appearance of a region of terrain.
+
+## Usage ##
+
+You assign a color and a normal texture by dragging texture assets to the **Color Texture** or **Normal Texture** fields, or by clicking {{< icon "file-folder.svg" >}}. Once the textures are assigned, you can adjust the appearance by using the **Normal Flip X** and **Normal Flip Y** toggles, and adjust the strength of the normals by using the **Normal Factor** slider. You can configure the dimensions of the region by adjusting the [Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape)
```
[the remark this answers is not in the document]
Comment (sphrose) [reply content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:12]: Fixed

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+
+[Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape)
+
+## Properties ##
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Color Texture** | The image that will be rendered on the terrain. | Texture | None |
+| **Normal Texture** | The texture that will be used as the normal map. | Texture | None |
+| **Normal Flip X** | Set to true to flip the normals about X. | Boolean | false |
```
[the remark this answers is not in the document]
Comment (sphrose) [reply content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:28]: Changed

File: content/docs/user-guide/components/reference/terrain/terrain-macro-material.md [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+| **Normal Flip Y** | Set to true to flip the normals about Y. | Boolean | false |
+| **Normal Factor** | Adjusts the strength of the normal values. | Float | 1.0 |
```
[the remark this answers is not in the document]
Comment (sphrose) [reply content/docs/user-guide/components/reference/terrain/terrain-macro-material.md:30]: Changed

**Review by FiniteStateGit — APPROVED** [review]
Looks good. Approved.

**Review by sptramer — APPROVED** [review]

**Comment by sptramer** [comment]
@invertednormal @mbalfour-amzn Will merge after tech review - only one tech reviewer required for merge.

**Comment by sptramer** [comment]
Merging with @FiniteStateGit grey-check approval.

**Pull request merged**